Update test to simplify command-line unit testing.

jzohrab opened the pull request:
With this change, tests can be run with 'python -m unittest test.unit.TestIOSXRDriver'.

dbarrosop commented on test/unit/TestIOSXRDriver.py:
```diff
@@ -57,7 +59,9 @@ class FakeIOSXRDevice:
     @staticmethod
     def read_txt_file(filename):
-        with open(filename) as data_file:
+        curr_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```
Why not just:

```
curr_dir = os.path.dirname(os.path.abspath(__file__))
```

jzohrab replied:
Because I didn't know such a thing!  I've always done it this way.  :-)  Learned something today, thanks @dbarrosop!

dbarrosop replied:
Ok, I was just asking in case there was some reason behind I was missing : )

jzohrab committed:
Update test to simplify command-line unit testing.

With this change, tests can be run with 'python -m unittest test.unit.TestIOSXRDriver'.

jzohrab commented:
@dbarrosop - I updated and rebased based on your feedback.  Thanks!
